docker-bin/pg/setup_oracle_fdw.sql

```diff
@@ -0,0 +1,379 @@
+-- Switch to Caseflow database for the environment you're in
+\c caseflow_certification_{ENV}
+-- Example in development:
+-- \c caseflow_certification_development
+
+-- Enable oracle_fdw extension
+CREATE EXTENSION oracle_fdw;
+
+-- This step is not required, however if you'd like to verify the extension was enabled
+-- you can print the extension version by uncommenting the command below
+-- SELECT oracle_diag ();
+
+-- Please note, if you are running the SQL commands below in the development environment
+-- the variables should be as follows
+-- ENV = development
+-- postgres_user = postgres
+-- local_host_port = 1521
+-- vacols_user = 'VACOLS_DEV'
+-- vacols_password = 'VACOLS_DEV'
+
+
+-- Skip if you aren't running this as part of an initialization script.
+-- To create the Caseflow Database, uncomment the line below
+-- CREATE DATABASE caseflow_certification_{ENV} OWNER {postgres_user} ENCODING UTF8;
+-- Example in development:
+-- CREATE DATABASE caseflow_certification_development OWNER postgres ENCODING UTF8;
+
+-- #####################
+-- Init foreign server
+-- #####################
+
+-- Create the foreign server
+-- Set the dbserver to point to the localhost on the port it uses to query VACOLS
+CREATE SERVER vacols_sv FOREIGN DATA WRAPPER oracle_fdw OPTIONS (
+    dbserver 'localhost:{local_host_port}/BVAP'
+);
```

Comment:
I was incorrect about the localhost piece - I was all tied up mentally at the time and my blunder was conflating how Caseflow accesses the VACOLS DB with our Postgres server will do so. I think it'd be best to leave the instructions more generic for the DBAs.

```suggestion
-- Set the dbserver to point to the URL of either the VACOLS DB instance or the
-- load balancer in front of it, along with the port.
CREATE SERVER vacols_sv FOREIGN DATA WRAPPER oracle_fdw OPTIONS (
    dbserver '{db_or_load_balancer_URL}:{oracle_listener_port}/BVAP'
);
```